count_word.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+
+import ("fmt"
+	"log"
+	"io/ioutil"
+	"os"
+	"bufio"
+	"sync"
+	"strings"
+	)
+
+
+func reader(filename string) []string {
+    file, err := os.Open(filename)
+    if err != nil {
+           log.Fatal(err)
+    }
+    scanner := bufio.NewScanner(file)
+    scanner.Split(bufio.ScanLines)
+    var txtlines []string
+
+    for scanner.Scan() {
+        txtlines = append(txtlines, scanner.Text())
+    }
+    defer file.Close()
```

Comment:
minor but I think you should move `defer` up to line 19, it'll be easier when reading